### PR TITLE
Remove GraphInstruction and the IRGen logic that it uses to generate TFE calls.

### PR DESCRIPTION
**Pull request by pschuh**
This is part of a larger set of changes to improve code health.

### CODE REVIEW

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

File: include/swift/AST/TensorFlow.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*
```
Comment (saeta) [inline include/swift/AST/TensorFlow.h]: Any reason not to remove the whole file?

**Review by saeta — APPROVED** [review]
I defer to @marcrasi in general, but this LGTM.

File: include/swift/AST/TensorFlow.h (hunk 1 above)
Comment (pschuh) [reply include/swift/AST/TensorFlow.h]: Ah, yes, I was being hacky when debugging the removal.

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Review by dan-zheng — APPROVED** [review]
LGTM (review requested).

**Pull request merged**